Makefile

```diff
@@ -24,7 +24,7 @@ tests-functional:
 	TEST_VERBOSE=false TEST_LOG_FORMAT=standard-verbose bash ./scripts/tests.sh -run TestFunctional
 
 tests-functional-verbose:
-	TEST_VERBOSE=true TEST_LOG_FORMAT=standard-verbose TEST_PARALLELISM=1 bash ./scripts/tests.sh -run TestFunctional
+	TEST_VERBOSE=true TEST_LOG_FORMAT=standard-verbose TEST_PARALLELISM=1 TEST_PARALLELISM_PKG=1 bash ./scripts/tests.sh -run TestFunctional
```

Comment:
Ak chcem spustit testy seriovo pre debug, tak to nefungovalo, ... ono su totiz 2 druhy paralelizmu testov:
- pocet testov, ktore bezia parallelne v jednom balicku (a sa pouzije `t.Parallel()`) - `TEST_PARALLELISM`
- pocet balickov, ktorych testy bezia paralelne - `TEST_PARALLELISM_PKG`